Lib/fontTools/merge.py

```diff
@@ -564,6 +572,50 @@ def merge(self, m, tables):
 def mapLookups(self, lookupMap):
 	pass
 
+
+# Gets unicode and font index from the composite gid
```

Comment:
I like you added comments describing function parameters, return value, etc. In Python these "docstrings" are usually placed below a function definition wrapped with `"""` quotes.